server/tests/follower.tests.js

```diff
@@ -0,0 +1,210 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import faker from 'faker';
+import models from '@models';
+import Token from '@helpers/Token';
+import app from '../app';
+
+chai.use(chaiHttp);
+const { expect } = chai;
+
+let firstUserToken, secondUserToken, firstUsername, secondUsername;
+const fakerObject = () => {
+  const username = faker.random.alphaNumeric(6);
+  const email = faker.internet.email();
+  const password = faker.internet.password();
+  return { username, email, password };
+};
+const userOne = fakerObject();
+const userTwo = fakerObject();
+
+describe('TESTS TO FOLLOW A USER', () => {
+  before(async () => {
+    const user1 = await models.User.create(userOne);
```

Comment:
move user generation into the factory (and call the needed functions from here)

Reply:
ok. implemented.